src/components/views/LocalModList.vue

```diff
@@ -436,6 +395,14 @@
                 Logger.Log(LogSeverity.ACTION_STOPPED, `${err.name}\n-> ${err.message}`);
             }
         }
+        
+        toggleMod(vueMod: any, enabled: boolean) {
+            if (enabled) {
+                this.enableMod(vueMod);
+            } else {
+                this.disableModRequireConfirmation(vueMod);
```

Comment:
Could be good to have it similar to uninstalling by checking the dependant count

Reply:
Isn't that what `disableModRequireConfirmation` is doing?